<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <include file="$(find anymal_wbc)/launch/spawn.launch">
    </include>

    <rosparam command="load" file="$(find anymal_wbc)/config/params.yaml" ns="anymal_wbc"/>
    <node name="wholeBodyControllerNode" pkg="anymal_wbc" type="wholeBodyControllerNode" output="screen"/>

</launch>